### ISSUE TITLE
Multiple finger involvement and Single finger involvement

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10694]
**Preferred term label:**
Two terms:
Multiple finger involvement
Single finger involvement

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Orthopedists use this distinction diagnostically because certain conditions appear in only one finger:

Multiple: Pyogenic flexor tenosynovitis
Single: Herpetic whitlow
Single: Felon

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001121
- label: Single-finger involvement
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality whose distribution and appearance are limited to or predominantly affect a single finger.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10694
- synonyms: Single finger involvement(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001122
- label: Multiple-finger involvement
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality whose distribution and appearance affect more than one finger.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10694
- synonyms: Multiple finger involvement(NONE)
